helm/k8s-nginx-ingress/templates/default-http-backend.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: default-http-backend
+  labels:
+    k8s-app: default-http-backend
+  namespace: {{ .Values.service.namespace }}
+spec:
+  replicas: 1
```

Comment:
This is the default - we might need to tweak later on, but all it does is 404 on any request to `/`